htdocs/kernel/object.php

```diff
@@ -13,14 +13,14 @@
 /**
  * This class is for compatibility with pre 2.6.0 code
  */
-abstract class XoopsObject extends Xoops\Core\Kernel\XoopsObject
```

Comment:
These should stay abstract. It should not be possible to directly instantiate any of these three classes.